keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -437,23 +451,28 @@ are always rejected.
 
 #### Well-known ClusterTrustBundles
 
-ClusterTrustBundles named with the prefix `kubernetes-io--` are reserved for
-future use by the Kubernetes project.
+Signer names with the domain prefix `kubernetes.io` (including any subdomains) are reserved by the Kubernetes project, and thus by extension the ClusterTrustBundle prefix name `kubernetes.io:` is reserved.
 
 When moving the ClusterTrustBundles feature to beta, we will figure out the
 semantics for exporting the trust anchors used by the built-in signers under
 this prefix.
```

Comment:
Beta is here, but I am unsure what this is saying.

Reply:
I think tactically we are going to have to punt this decision to 1.31, since we're about to hit the enhancements freeze.  It would be nice to get the API types themselves to beta in 1.30.